app/src/src/screens/menu/ProfileScreen.jsx

```diff
@@ -126,9 +141,20 @@ const styles = StyleSheet.create({
     flex: 1,
     backgroundColor: '#eeeeee',
   },
+  contentContainer: {
+    paddingBottom:
+      Platform.OS === 'android' ? '40%' : Platform.OS === 'ios' && '30%',
+  },
+  scrollViewStyle: {
+    flex: 1,
+    marginHorizontal: 1,
+    height: '100%',
+    width: '100%',
+    alignSelf: 'center',
+  },
   backHeading: {
     width: '100%',
-    height: '18%',
+    height: Platform.OS === 'android' ? '18%' : Platform.OS === 'ios' && '30%',
```

Comment:
What will the height be when the platform is neither android or web?

Reply:
I changed all of them to check for android first and then have the second choice be anything else (ios, web, etc) because I'm not sure how it'll look on web.